spec.html

```diff
@@ -9908,7 +9908,7 @@ <h2>Syntax</h2>
   <!-- es6num="11.3" -->
   <emu-clause id="sec-line-terminators">
     <h1>Line Terminators</h1>
-    <p>Like white space code points, line terminator code points are used to improve source text readability and to separate tokens (indivisible lexical units) from each other. However, unlike white space code points, line terminators have some influence over the behaviour of the syntactic grammar. In general, line terminators may occur between any two tokens, but there are a few places where they are forbidden by the syntactic grammar. Line terminators also affect the process of automatic semicolon insertion (<emu-xref href="#sec-automatic-semicolon-insertion"></emu-xref>). A line terminator cannot occur within any token except a |StringLiteral|, |Template|, or |TemplateSubstitutionTail|. Line terminators may only occur within a |StringLiteral| token as part of a |LineContinuation|.</p>
+    <p>Like white space code points, line terminator code points are used to improve source text readability and to separate tokens (indivisible lexical units) from each other. However, unlike white space code points, line terminators have some influence over the behaviour of the syntactic grammar. In general, line terminators may occur between any two tokens, but there are a few places where they are forbidden by the syntactic grammar. Line terminators also affect the process of automatic semicolon insertion (<emu-xref href="#sec-automatic-semicolon-insertion"></emu-xref>). A line terminator cannot occur within any token except a |StringLiteral|, |Template|, or |TemplateSubstitutionTail|. &lt;LF&gt; and &lt;CR&gt; line terminators cannot occur within a |StringLiteral| token except as part of a |LineContinuation|.</p>
```

Comment:
For clarity and future explorers, can you elaborate on why this restriction is narrowed?

Reply:
It's the fundamental change of [proposal-json-superset](https://github.com/tc39/proposal-json-superset), just in a part of the spec that I didn't realize included affected text until working on https://github.com/tc39/test262/pull/1565 . As of this PR landing, _StringLiteral_ will be able to include any _LineTerminator_ as part of a _LineContinuation_ (backslash followed by _LineTerminatorSequence_, used to statically define long strings over multiple lines without affecting their content), and—invalidating the old text—will **also** be able to include U+2028 LINE SEPARATOR and U+2029 PARAGRAPH SEPARATOR line terminators in any position (as is the case for JSON strings).

Reply:
Previously, no line terminators could occur in string literals (except as part of a `LineContinuation`).

With the change, U+2028 and U+2029 can occur in string literals, even outside of `LineContinuation`.

In other words, this change clarifies that `LineContinuation`s are unaffected by the change, i.e. `\` followed by U+2028 or U+2029 in a string literal still evaluates to the empty string. This was already clear from the proposal, but updating this note as part of this patch improves accuracy.